contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -304,7 +304,7 @@ contract VaultAdmin is VaultStorage {
      */
     function harvest(address _strategyAddr)
         external
-        onlyVaultOrGovernor
+        onlyVaultOrGovernorOrStrategist
```

Comment:
We should apply the same modifier to the other harvest method above (the one without a strategy address).

BTW, I'm not sure why currently those 2 methods have different modifiers? onlyGovernor for one, onlyVaultOrGovernor for the second. I think it was probably an oversight and we forgot to update both?

Reply:
The VaultCore makes an external call into the VaultAdmin to call the harvest(address) method, which is why harvest(address) has the additional vault access permission.

Reply:
Ah yes makes sense.